Automattic/jetpack
# Subscription form: adding a default line-height to avoid differences using different font-faces on the input and button elements

**The issue consists of the height of the input and button elements doesn't match.**
When a theme has more than one font face available and the Jetpack subscription form renders the email input element and the form button using different font faces we can reproduce the issue. Example using Twenty Twenty One theme:

![image](https://user-images.githubusercontent.com/1310626/147290733-d8bb191b-d646-4ac5-8cc2-c288b94cfb4f.png)
**Why one element is taller than the other?**
Because 
- The final computed line height can vary depending on the font face rendered
- Browsers apply a minimum line height to certain elements.
The result, in this case, is that the line height computed for the button is larger than the one computed for the email input, so the button ends up being larger than the input.

#### Changes proposed in this Pull Request:

In this PR I'm changing the current line-height value for these elements. Why? Because the current value, `initial`, defaults to `normal`, and then the computed value can be different between the input element and the button element based on the font face rendered. 
```
Normal: Tells user agents to set the used value to a "reasonable" value based on the font of the element. The value has the same meaning as <number>. We recommend a used value for 'normal' between 1.0 to 1.2. The computed value is 'normal'. 
```

But if we use a unitless number we can get an equal line height based on the font size of the elements (which is equal for both the input and the button).
```
Number: The used value of the property is this number multiplied by the element's font size. Negative values are illegal. The computed value is the same as the specified value.
```
This number should be a little bit larger than the default line-height (seems to be around 1.2) otherwise we can't be sure that the computed line-height will be equal for both elements.
[Reference](https://www.w3.org/TR/CSS2/visudet.html#propdef-line-height)


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






1. Insert a Jetpack subscription block in the WordPress editor
2. Change the typeface options for your theme
3. Navigate to the public view of the post with the subscription block.
4. Check the size of both elements not only visually but using the browser inspector measure tools.

Before:
![image](https://user-images.githubusercontent.com/1310626/147292729-1f1680ca-7590-4bd2-b32e-45ce9b0e9d16.png)

After:
![image](https://user-images.githubusercontent.com/1310626/147292444-fbc0a76d-6ff7-49ca-99f4-249f7121da67.png)

Issues Related:
https://github.com/Automattic/themes/issues/2488
https://github.com/Automattic/jetpack/issues/17590
